src/Results/Components/Results.tsx

```diff
@@ -21,58 +20,54 @@ const mapStateToProps = (state: any) => {
+  if (vaccineResults.length > 0) {
+    displayResult = (
       <>
-        <div className="viewScreen">
-          <div className="wrapper">
-            <TableContainer>
-              <Table aria-label="simple table">
-                <TableHead>
-                  <TableCell className="sticky-col1">
-                    Vaccination Center
-                  </TableCell>
-                  <TableCell className="sticky-col2">
-                    Minimum Age Limit
-                  </TableCell>
-                  <TableCell align="left">Date</TableCell>
-                  <TableCell>Available Slots</TableCell>
-                  <TableCell>Vaccine Type</TableCell>
-                  <TableCell>Dose1 Slots</TableCell>
-                  <TableCell>Dose2 Slots</TableCell>
-                </TableHead>
-                <TableBody>
-                  {data.map((element: any) =>
-                    element.sessions.map((data: any) => {
-                      return (
-                        <TableRow>
-                          <TableCell className="sticky-col first-col">
-                            {element.name}
-                          </TableCell>
-                          <TableCell className="sticky-col second-col">
-                            {data.min_age_limit}
-                          </TableCell>
-                          <TableCell align="center">{data.date}</TableCell>
-                          <TableCell align="center">
-                            {data.available_capacity}
-                          </TableCell>
-                          <TableCell>{data.vaccine}</TableCell>
-                          <TableCell align="center">
-                            {data.available_capacity_dose1}
-                          </TableCell>
-                          <TableCell align="center">
-                            {data.available_capacity_dose2}
-                          </TableCell>
-                        </TableRow>
-                      );
-                    })
-                  )}
-                </TableBody>
-              </Table>
-            </TableContainer>
-          </div>
+        <div className="tableWidth">
+          <TableContainer>
+            <Table aria-label="simple table">
+              <TableHead>
+                <TableCell className="sticky-col1">
+                  Vaccination Center
+                </TableCell>
+                <TableCell className="sticky-col2">Minimum Age Limit</TableCell>
+                <TableCell align="left">Date</TableCell>
+                <TableCell>Available Slots</TableCell>
+                <TableCell>Vaccine Type</TableCell>
+                <TableCell>Dose1 Slots</TableCell>
+                <TableCell>Dose2 Slots</TableCell>
+              </TableHead>
+              <TableBody>
+                {vaccineResults.map((element: any) =>
+                  element.sessions.map((data: any) => {
+                    return (
+                      <TableRow>
```

Comment:
It might make sense to give key property for TableRow https://reactjs.org/docs/lists-and-keys.html